apps/ui/src/components/FormSpaceLabels.vue

```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
+import { SpaceMetadataLabel } from '@/types';
+
+const labels = defineModel<SpaceMetadataLabel[]>({
+  required: true
+});
+
+const { addNotification } = useUiStore();
+const modalOpen = ref(false);
+const activeLabelId = ref<string | null>(null);
+
+function toggleModal(open: boolean = false, labelId: string | null = null) {
+  modalOpen.value = open;
+  activeLabelId.value = labelId;
+}
```

Comment:
```suggestion
function setModalStatus(open: boolean = false, labelId: string | null = null) {
  modalOpen.value = open;
  activeLabelId.value = labelId;
}
```

I'd call it something else as toggle makes it sound like it always toggle to opposite of previous state, but we actually control it ourselves.